www/package.json

```diff
@@ -34,13 +34,21 @@
     "@mdx-js/mdx": "^1.0.0-rc.4",
     "@mdx-js/react": "^1.0.0-rc.5",
     "@rhythm-ui/button-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/expand-collapse-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/grid-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/layout-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/rui-breadcrumbs-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/rui-card-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/skip-links-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
     "@rhythm-ui/story-react": "^1.0.0",
+"@rhythm-ui/scroll-to-react": "^1.0.0",
```

Comment:
1 import would seem sufficient here ;)

Reply:
@ricominten Oh dear, I don't know what happened there! Could be a merge issue or possible a bug with the generator logic? I'm not sure what would cause that